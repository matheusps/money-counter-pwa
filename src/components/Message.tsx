import styled from 'styled-components'

const Message = styled.h2`
  color: ${props => props.theme.dark};
`

export default Message
