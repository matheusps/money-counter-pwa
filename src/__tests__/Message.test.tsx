import React from 'react'
import { render } from 'react-testing-library'
import 'jest-styled-components'

import Message from '../components/Message'

describe('Message component', () => {
  const component = render(<Message>Testing</Message>)

  it('should render', () => {
    expect(component).toBeTruthy()
  })

  it('should match snapshot', () => {
    expect(component.asFragment()).toMatchSnapshot()
  })
})
