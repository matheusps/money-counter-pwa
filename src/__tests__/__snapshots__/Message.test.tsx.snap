// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message component should match snapshot 1`] = `
<DocumentFragment>
  <h2
    class=""
  >
    Testing
  </h2>
</DocumentFragment>
`;
